spec/support/saml_response_helper.rb

```diff
@@ -34,6 +34,13 @@ def raw_xml_response
       end
     end
 
+    def saml_response(settings)
+      @_saml_response ||= OneLogin::RubySaml::Response.new(
+        raw_xml_response,
+        settings: settings
+      )
+    end
```

Comment:
There is another `saml_response` method in this file. Can that one be removed now that this one has been added?